### ISSUE TITLE
Feature request: Custom CSS

### ISSUE THREAD

**Issue by shvchk** [conceptualspace/yet-another-speed-dial#74]

**Comment by conceptualspace** [comment]
agreed. i was thinking about providing some different themes first, and then it shouldn't be too much to enable this for the more advanced users. im curious what you would change, if you put together some nice overrides that could be the basis of a theme i'm open to a PR!

**Comment by Athari** [comment]
1. For me, the main problem with the current design is that folder links require pixel hunting half a screen away from the dials. So the first thing I'm going to do when support for custom CSS is implemented is replacing margins with padding on folder links and moving them to the middle.

2. Another thing is displaying all folders at once. It feels like too much space is wasted. It'd obviously be a hack, so if nested folders are implemented (#36), it becomes unnecessary.

3. And while I like the minimalistic design, some sort of hover effect won't hurt. Just some image brightness and text color, nothing fancy.

Now that I think about it, (1) and (3) can be a part of the base theme, but I don't know whether you'd be interested in that.
